tests/KdybyTests/Doctrine/Connection.phpt

```diff
@@ -34,7 +34,10 @@ class ConnectionTest extends Tester\TestCase
 	protected function loadMysqlConfig()
 	{
 		$configLoader = new Nette\DI\Config\Loader();
-		$config = $configLoader->load(__DIR__ . '/../../mysql.neon', isset($_ENV['TRAVIS']) ? 'travis' : 'localhost');
+		$config = $configLoader->load(__DIR__ . '/../../mysql.neon');
+		if (!isset($_ENV['TRAVIS'])) {
```

Comment:
I like it, but I would make it dependent on the existence of the file, not `_ENV` variable